src/build_system.ml

```diff
@@ -334,6 +335,7 @@ module Alias0 = struct
   let doc         = make "doc"
   let private_doc = make "doc-private"
   let lint        = make "lint"
+  let all         = make "all"
```

Comment:
there's a bit of duplication between here and `is_standard`. What do you think about building a list ref in `make` (calling it `register` maybe?) and have `is_standard` implemented on top of it?

Reply:
This bugs me as well. I'll do it in a separate PR if you don't mind as it has not relevance to this particular alias.

Reply:
absolutely.